src/api-reference/request/v4.get-started.markdown

```diff
@@ -3,7 +3,7 @@ title: Travel Request v4 - Get Started
 layout: reference
 ---
 
-{% include prerelease.html %}
+** Pre-release Version 4.0**
```

Comment:
This is a regression from current documentation which uses the `includes` file for consistency across all docs.